client/client.go

```diff
@@ -46,10 +46,11 @@ type Client struct {
 
 // Configuration stores the live configuration and global parameters of a client
 type Configuration struct {
-	API     ApiConf    // location of the MIG API
-	Homedir string     // location of the user's home directory
-	GPG     GpgConf    // location of the user's secring
-	Targets TargetConf // Target macro specification
+	API      ApiConf      // location of the MIG API
+	Homedir  string       // location of the user's home directory
+	GPG      GpgConf      // location of the user's secring
+	Targets  TargetConf   // Target macro specification
+	Terminal TerminalConf // Terminal configuration
```

Comment:
Seems like this would always just remain a single option, maybe we don't need a new section for it in that case and just include it in the main configuration?